developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_27.rst

```diff
@@ -16,16 +16,23 @@ Make sure your ``appinfo/info.xml`` allows for Nextcloud 27.
 .. code-block:: xml
 
     <dependencies>
-        <nextcloud min-version="23" max-version="27" />
+        <nextcloud min-version="25" max-version="27" />
     </dependencies>
 
 Front-end changes
 -----------------
 
-Removed APIs
-^^^^^^^^^^^^
+Added APIs
+^^^^^^^^^^
+
+* Files Router API https://github.com/nextcloud/server/pull/37824
```

Comment:
needs detailed docs

Reply:
Was already mentioned in https://github.com/nextcloud/server/pull/37824#pullrequestreview-1393369039

So something @skjnldsv needs to provide and then backport to stable27 in the docs as well.
But we should not let this block app devs getting informed about all the other changes, agreed?

Reply:
https://github.com/nextcloud/documentation/pull/10428